ai-decision-services/ai-ad-dpp-engine/setup-event/setup-event.md

```diff
@@ -0,0 +1,69 @@
+Lab 7: Set up Events
+===
+
+OCI event is a service listening to desired change happening in the target service. Here is a list of [DB](https://docs.oracle.com/en-us/iaas/Content/Events/Reference/eventsproducers.htm#dbaasevents__AutoDB) and [Object Storage](https://docs.oracle.com/en-us/iaas/Content/Events/Reference/eventsproducers.htm#ObjectStor__bucket) related events that Event can monitor.
```

Comment:
Reword -
OCI Events tracks resource changes with events which comply with CNCF CloudEvents Standard.  Use the links below to learn more about ADB and OCI Object Storage events which OCI Events Service can monitor.